roles/openshift_metrics/files/templates/cassandra_serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
so in logging... template files are found in <role>/templates  instead of <role>/files/templates.  this way you can specify src=template_name without having to define explicitly where it is located.